src/todo-list.js

```diff
@@ -0,0 +1,134 @@
+class List {
+  constructor() {
+    this.list = JSON.parse(localStorage.getItem('todolist'));
+    if (!this.list) {
+      this.list = [];
+    }
+    this.display();
+  }
+
+  display() {
+    this.saveData();
+    const listSection = document.querySelector('.items');
+    listSection.innerHTML = '';
+    this.list.forEach((work) => {
+      let taskItem = `
+     <li class="list-items" id="item-data-${work.index}">`;
+      if (work.completed) {
+        taskItem += `
+            <span class="material-icons done update-status" data="${work.index}">
+              done
+            </span>
+            <p contenteditable="true" class="completed work" data="${work.index}">
+              ${work.description}
+            </p>
+            `;
+      } else {
+        taskItem += `
+            <span class="material-icons  update-status"  data="${work.index}">
+              check_box_outline_blank
+            </span>
+            <p contenteditable="true" class="work" data="${work.index}">
+              ${work.description}
+            </p>`;
+      }
+      taskItem += `
+          <span class="material-icons delete-work" data="${work.index}">
+            delete
+          </span>
+        </li>
+      `;
+      listSection.innerHTML += taskItem;
+    });
+    this.activateActions();
+  }
+
+  // add new activity
+  addWork(work) {
+    if (work || work === 0) {
+      const newWork = {
+        description: work,
+        completed: false,
+        index: this.list.length,
+      };
+      this.list.push(newWork);
+      this.display();
+    }
+  }
+
+  updateActivityStatus(workIndex) {
+    if (workIndex !== undefined) {
+      if (this.list[workIndex].completed === true) {
+        this.list[workIndex].completed = false;
+      } else {
+        this.list[workIndex].completed = true;
+      }
+    }
+    this.display();
+  }
+
+  deleteWork(workIndex) {
+    if (workIndex) {
+      this.list.splice(workIndex, 1);
+      this.display();
+    }
+  }
```

Comment:
- [ ] After deleting todo index should update in local storage. Also, that's why if ı delete todo-2 ı cant click on todo-3 in the list. Please fix it. [Requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Deleting%20a%20task%20should%20update%20all%20remaining%20items%27%20indexes%2C%20so%20they%20represent%20the%20current%20list%20order%20and%20are%20unique(i.e.%20if%20you%27re%20deleting%20the%20first%20task%20index%201%20from%20the%20list%2C%20the%20index%20of%20the%20next%20task(2)%20should%20set%20to%201)..)
![image](https://user-images.githubusercontent.com/38635158/166965011-dc83fd7a-adec-45d7-b862-03a20f5326a2.png)